uint16 planeID

bool need_idel

bool need_take_off

bool need_mission

bool need_land

bool need_formation

bool need_protected

uint8 swarm_shape

uint8 One_column=0
uint8 Triangle=1
uint8 One_row=2
